tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/variabledef/ErrorVariableDefinitionTest.java

```diff
@@ -238,7 +238,7 @@ public void testNegativeErrorVariables() {
         BAssertUtil.validateError(resultNegative, ++i,
                 "redeclared symbol 'message'", 54, 26);
         BAssertUtil.validateError(resultNegative, ++i,
-                "incompatible types: expected 'int', found 'map<(anydata|error)>'", 56, 18);
+                "incompatible types: expected 'int', found 'map<(anydata|error|string)>'", 56, 18);
```

Comment:
Identifying the narrowest type will remove the requirement to add `string` here.

Reply:
Fixed above